src/main/java/bio/terra/app/controller/SearchApiController.java

```diff
@@ -47,7 +48,7 @@
   private final AuthenticatedUserRequestFactory authenticatedUserRequestFactory;
   private final SearchService searchService;
   private final SnapshotService snapshotService;
-  private final SnapshotSearchMetadataDao snapshotMetadataDao;
```

Comment:
I've been thinking about the repetition we have in our class names vs the package names. It does seem redundant to call the DAO `SnapshotSearchMetadataDao` when it's in package `search`. But we do have this convention so I thought I could drop it from the variables at least, as it doesn't add any useful information to the name. But that practice doesn't follow our conventions either...